examples/sandbox/package.json

```diff
@@ -9,12 +9,9 @@
     "@elastic/search-ui-app-search-connector": "1.0.0",
     "@elastic/search-ui-site-search-connector": "1.0.0",
     "moment": "^2.24.0",
-    "react": "^16.8.1",
-    "react-dom": "^16.8.1",
-    "react-scripts": "2.1.5"
-  },
-  "devDependencies": {
-    "babel-eslint": "9.0.0"
```

Comment:
just curious - why remove this? is it because it's already a nested dependency of create-react-app?

Reply:
Exactly, it will error out otherwise. I think we added it explicitly at one time to overcome some error or another, but it's no longer necessary.